processor/transformprocessor/internal/common/functions.go

```diff
@@ -140,6 +143,54 @@ func limit(target GetSetter, limit int64) (ExprFunc, error) {
 	}, nil
 }
 
+func replaceMatch(target GetSetter, pattern string, replacement string) (ExprFunc, error) {
+	_, err := filepath.Match(pattern, "")
```

Comment:
Could be for an optimizing followup, but we probably want to use the `glob` library to have a compiled matcher as this will probably be quite a bit slower. One of the big reasons these are factories is to allow compiling regex or similar. It's already used in the collector e.g.

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/abba937839cbc273a26ea3bf554c6f7ec1a2f25f/internal/docker/matcher.go#L22

Reply:
I will look into this.

Reply:
This is exactly what I was looking for in standard libraries, but couldn't find. Thanks for pointing it out this package.  Updated to use glob.